docs/core/native-interop/expose-components-to-com.md

```diff
@@ -80,14 +80,24 @@ Open an elevated command prompt and run `regsvr32 ProjectName.comhost.dll`. That
 
 The resulting output will now also have a `ProjectName.X.manifest` file. This file is the side-by-side manifest for use with Registry-Free COM.
 
+## Embedding Type Libraries in the COM host
+
+Unlike in .NET Framework, there is no support in .NET Core or .NET 5 or newer for generating a COM Type Library (TLB) from a .NET Core assembly. The guidance is to either manually write an IDL file or a C/C++ header for the native declarations of the COM interfaces. If you decide to write an IDL file, you can compile it with the Visual C++ SDK's MIDL compiler to produce a TLB.
+
+In .NET 6 Preview 5 or newer, the .NET SDK supports embedding already-compiled TLBs into the COM host as part of your project build.
```

Comment:
It is only the .NET SDK that needs to be .NET 6 Preview 5 or newer, right? i.e. using a .NET 6 SDK to build a COM component targeting .NET 5 (or .NET Core 3.1) will be able to embed TLBs in the older comhost?